torchaudio/backend/_soundfile_backend.py

```diff
@@ -11,6 +11,42 @@
     import soundfile
 
 
+# Mapping from soundfile subtype to number of bits per sample.
+# This is mostly heuristical and the value is set to 0 when it is irrelevant
+# (lossy formats) or when it can't be inferred.
+# The dict is inspired from
+# https://github.com/bastibe/python-soundfile/blob/744efb4b01abc72498a96b09115b42a4cabd85e4/soundfile.py#L66-L94
```

Comment:
Can you add references you looked at, for formats that are not immediately clear like ADPCMs? So that later, it's easy for other maintainer to look at it.